core/src/main/java/org/mskcc/cbio/oncokb/util/IndicatorUtils.java

```diff
@@ -29,6 +29,16 @@ public static IndicatorQueryResp processQuery(Query query, String geneStatus,
             return indicatorQuery;
         }
 
+        // Set the alteration to empty string in order to get relevant variants.
+        if (query.getAlteration() == null) {
+            query.setAlteration("");
+        }
+
+        // Need to find a better way to map the alteration name with the variant consequence
+        if (query.getAlteration().isEmpty() && query.getConsequence().equals("splice_region_variant")) {
+            query.setAlteration("splice mutation");
+        }
+
         if (query.getAlteration() != null && query.getAlteration().toLowerCase().matches("gain")) {
```

Comment:
query.getAlteration() != null checking might not be necessary any more, since you set it to empty when it is null at line 33.